src/cli/index.ts

```diff
@@ -63,6 +63,7 @@ function printUsage() {
 const signOptions = {
   oidcClientID: 'sigstore',
   oidcIssuer: 'https://oauth2.sigstore.dev/auth',
+  oidcRedirectURL: process.env.OIDC_REDIRECT_URL,
```

Comment:
Do we want to set this using env or should it be a [flag like cosign](https://github.com/sigstore/cosign/blob/ce8177ce72c54bfa55b8de2e65a97d3e608ba4f1/doc/cosign_sign.md?plain=1#L88)? 🤔 Also found [this env var in gitsign](https://github.com/sigstore/gitsign/blob/2509c5c5f931b5de2cc9ca2449d7e309a65a2ec9/internal/config/config_test.go#L71).

Reply:
Yeah good point, I like this idea of publishing the CLI as a separate package. Would also mean we start from scratch on the interface.